src/components/Calculator/Calculator.jsx

```diff
@@ -1,115 +1,173 @@
+			.then(() => {
+				if (!deposits.error && !deposits.isLoading) {
+					navigate('/deposits');
+				}
+			})
+			.catch(() => {
+				alert(deposits.message);
+			});
 	};
 
-	const getCurrencyValue = (value) => {
-		setCurrency(value);
+	const roundNumber = (number) => {
+		return parseFloat(Math.round(number)).toLocaleString();
+	};
+
+	const replacePointNumber = (number) => {
+		return String(number).replace('.', ',');
 	};
 
 	return (
 		<section className="calculator">
-			<h2 className="calculator__title title">калькулятор</h2>
+			<h2 className="calculator__title title">Калькулятор</h2>
 			<div className="calculator__container">
 				<div className="calculator__products">
-					<span className="calculator__product" onClick={chooseDeposit} role="presentation">
-						вклады
+					<span
+						className={`calculator__product ${
+							!calculator.isCredit && 'calculator__product_active'
+						}`}
+						onClick={chooseDeposit}
+						role="presentation"
+					>
+						Вклады
 					</span>
-					<span className="calculator__product" onClick={chooseCredit} role="presentation">
-						кредиты
+					<span
+						className={`calculator__product ${calculator.isCredit && 'calculator__product_active'}`}
+						onClick={chooseCredit}
+						role="presentation"
+					>
+						Кредиты
 					</span>
 					<div className="calculator__slider" ref={sliderRef} />
 				</div>
-				<form>
+				<form onSubmit={handleSubmit}>
 					<div className="calculator__calculation">
 						<div className="calculator__items">
-							{!isCredit ? (
+							{!calculator.isCredit && (
 								<>
-									<Range
-										name="summ"
-										placeHolder="cумма вклада"
-										symbol={currency}
-										min={10000}
-										max={1000000}
-										startValue={240000}
-									/>
 									<Select
-										getValue={getCurrencyValue}
-										name="currency"
-										placeHolder="валюта"
-										options={['₽', '$']}
+										name="depositAmount"
+										placeHolder="Сумма вклада"
+										currency={['₽', '$', '€', '¥']}
+										defaultValue={calculator.depositAmount}
+										getValue={getValues}
+										max="10000000"
+										min="15000"
 									/>
 									<Range
-										name="term"
-										placeHolder="срок в месяцах"
-										min={1}
-										max={360}
-										startValue={60}
+										name="depositTerm"
+										placeHolder="Срок в месяцах"
+										min="1"
+										max="36"
+										startValue={calculator.depositTerm}
+										getValue={getValues}
 									/>
 								</>
-							) : (
+							)}
+							{calculator.isCredit && (
 								<>
+									<Select
+										name="creditAmount"
+										placeHolder="Сумма кредита"
+										currency={['₽', '$', '€', '¥']}
+										defaultValue={calculator.creditAmount}
+										getValue={getValues}
+										max="10000000"
+										min="15000"
+										disableOption={true}
```

Comment:
У булевх props можно передавать только название
```suggestion
										disableOption
```